Source/Core/Core/PowerPC/SignatureDB/SignatureDB.h

```diff
@@ -18,6 +18,51 @@ class SignatureDBFormatHandler;
 class SignatureDB
 {
 public:
+  enum class Handler
+  {
+    DSY,
+    CSV
+  };
+  SignatureDB(Handler handler = Handler::DSY);
```

Comment:
These constructors should be `explicit`. There's also no real need to make this a default argument.